### PR TITLE
Commenting code rough draft (For classes):

### PR DESCRIPTION
**Pull request by dominic-diblasio**
## Initial pull request - Coding Conventions!
-Adding TeamConventions.java to our project
-Adding reference to TeamConventions class in Main.java

*We will further expand on the decisions and types of formatting that we'd like to use when we meet!

### CODE REVIEW

File: src/main/java/main/TeamConventions.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    /// [THIS IS A HEADER] (Group methods or systems) ///
+
+    /**
+     * <b>This is a template explanation of a method or other implementation</b>
+     * <br><br>
+     * Description: You'll also notice that hovering over the name of the
+     * method, or any other line including the method name,
+     * it will include this explanation!
+     */
```
Comment (jjszaniszlo) [inline src/main/java/main/TeamConventions.java:32]: This is a great explanation and I see that you are using javadoc semantics for the formatting.  Maybe we can have an example for parameters, return type, etc.  That way we can start off with amazing documentation with no effort ;)

**Review by jjszaniszlo — COMMENTED** [review]
great work so far! Unless you want to test the javadoc stuff, I will let you continue working on it before merging 🚀

File: src/main/java/main/TeamConventions.java (hunk 1 above)
Comment (storygames-ddiblasio) [reply src/main/java/main/TeamConventions.java:32]: I am 100% aligned with your idea -- and thanks for the thorough check!

Comment (dominic-diblasio) [reply src/main/java/main/TeamConventions.java:32]: I am 100% aligned with your idea -- and thanks for the thorough check!

Comment (edwardeckelberry) [reply src/main/java/main/TeamConventions.java:32]: not familiar with github so it'll take a little bit for me to get used to, but great job!

Comment (dominic-diblasio) [reply src/main/java/main/TeamConventions.java:32]: Thanks for reviewing Eddie!

**Pull request merged**